modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromRegFactory.pm

```diff
@@ -15,39 +15,91 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
-=cut
-
-=pod
-
 =head1 NAME
 
 Bio::EnsEMBL::Compara::RunnableDB::PrepareMaster::UpdateGenomesFromRegFactory
 
-=head1 SYNOPSIS
+=head1 DESCRIPTION
 
 Returns the list of species/genomes to add to the master database from the core
 databases in the registry file
 
+=head1 EXAMPLES
+
+    standaloneJob.pl Bio::EnsEMBL::Compara::RunnableDB::PrepareMaster::UpdateGenomesFromRegFactory \
+        -compara_db $(mysql-ens-compara-prod-9-ensadmin details url jalvarez_prep_vertebrates_master_for_rel_103) \
+        -master_db $(mysql-ens-compara-prod-1 details url ensembl_compara_master)
+
 =cut
 
 package Bio::EnsEMBL::Compara::RunnableDB::PrepareMaster::UpdateGenomesFromRegFactory;
 
 use warnings;
 use strict;
+
 use Bio::EnsEMBL::Registry;
+use Bio::EnsEMBL::Compara::Utils::CoreDBAdaptor;
 
 use base ('Bio::EnsEMBL::Hive::Process');
 
+
```

Comment:
I don't think we need this new newline
```suggestion
```

Reply:
It's a Python PEP8 thingy that has gotten stuck in my brain :sweat_smile: